module/data/actor/npc.mjs

```diff
@@ -1,10 +1,15 @@
+import ChallengeFields from "./templates/challenge.mjs";
+import ActorDescriptionTemplate from "./templates/description.mjs";
 import NamegiverTemplate from "./templates/namegiver.mjs";
 
+
 /**
  * System data definition for NPCs.
  * @mixin
  */
-export default class NpcData extends NamegiverTemplate {
+export default class NpcData extends NamegiverTemplate.mixin(
+    ActorDescriptionTemplate
```

Comment:
If this is in here too, then it should be mixed in with the NamegiverTemplate directly, since PC and NPC are the only one extending the namegiver template anyways.

Reply:
i added it to the common template, since all actors have a description